.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,28 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ruby
+{
+  "name": "Ruby 2.5 on Debian",
+  "image": "ruby:2.5-slim-buster", // Specify the Ruby 2.5 image
+  "postCreateCommand": "RUBYGEMS_VERSION=3.3.27\ngem install rubygems-update -v ${RUBYGEMS_VERSION} > /dev/null 2>&1\nupdate_rubygems > /dev/null 2>&1\n", // Optional: Run this command after the container is created
+
+  // Features to add to the dev container. More info: https://containers.dev/features.
+  "features": {
+    "./apt-install": {}
+  },
```

Comment:
Do we need this?
It seems that `install.sh` installs needless packages.

Reply:
I think that all packages are needless because:
* We don't need to build CRuby (`ruby:2.5-buster` already has built CRuby)
* We don't need JRuby (We should use separated `jruby` image instead)

BTW, how can I test the current configuration on local...? I'm not a Visual Studio Code user. So I want to test without Visual Studio Code...

Reply:
I have only done it from within RubyMine (in which case you just right click on the file to start the container with a local RubyMine inside the container and a cross mounted project).

But this is how I would do it from the CLi:

https://containers.dev/supporting#devcontainer-cli

Reply:
I will remove the apt install script and see if that works.

Reply:
I've modified the feature to setup bundler instead of installing apt packages.
```
root@d5a0bdd30fe3:/IdeaProjects/rexml# bundle -v
Bundler version 2.3.27
root@d5a0bdd30fe3:/IdeaProjects/rexml# bundle exec rake
/usr/local/bin/ruby test/run.rb
Loaded suite test
Started
Finished in 4.109169655 seconds.
-----------------------------------------------------------------------------------------------------------
716 tests, 2170 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed
-----------------------------------------------------------------------------------------------------------
174.24 tests/s, 528.09 assertions/s
```

Reply:
In doing so I noticed that this gem has bundler as a dependency in the Gemfile, and that's not a good idea, unless the gem is literally something that hacks on bundler.  This gem only uses bundler in the normal, vanilla, way, so it should not have bundler, even as a development dependency. Added as #276 

@kou This PR is ready for review.